packages/react-native-performance/src/performance-entry.ts

```diff
@@ -34,13 +65,16 @@ export class PerformanceMark extends PerformanceEntry {
 }
 
 export class PerformanceReactNativeMark extends PerformanceEntry {
-  constructor(name, startTime) {
+  constructor(name: string, startTime: number) {
     super(name, 'react-native-mark', startTime, 0);
   }
 }
 
 export class PerformanceMetric extends PerformanceEntry {
-  constructor(name, metricOptions = {}) {
```

Comment:
I think it is better to not add this default. Empty object means every parameter in metricOptions optional but it shouldn't be imo. If metric option doesn't have a value and startTime it shouldn't be logged. What do you think?

Reply:
sure